documentation/bolt_inventory_reference.md

```diff
@@ -0,0 +1,376 @@
+# `inventory.yaml` options
+
+The [inventory file](inventory_file_v2.md) is used to store information about
+your targets and arrange them into groups. Both targets and groups can
+include data such as facts or vars and can configure the transports Bolt uses
+to establish connections with targets.
+
+## Options
```

Comment:
```suggestion
## Top-level options

The top level of your inventory file contains options that configure the implicit `all` group. These options apply to all of the groups and targets and targets in the inventory file. For more information on inventory files, such as precedence, see [Inventory files](inventory_file_v2.md)
```
The next heading is "Group objects", but obviously contains some identical options. We need something to make it clearer that all of these options are for the `all` group. I think top-level is fine, but I'm open to suggestions. I think adding the short intro paragraph is also helpful

Reply:
These are more like _fields_ than _options_ because this file is more akin to data than configuration.